namespaces/live.cloud-platform.service.justice.gov.uk/offender-categorisation-dev/06-certificates.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: dev.offender-risk-profiler.service.justice.gov.uk
+  namespace: offender-categorisation-dev
+spec:
+  secretName: offender-risk-profiler-cert
+  issuerRef:
+    name: letsencrypt-production
+    kind: ClusterIssuer
+  dnsNames:
+  - dev.offender-risk-profiler.service.justice.gov.uk
```

Comment:
These DNS don't follow convention, should be `offender-risk-profiler-dev.hmpps.service.justice.gov.uk`